elasticsearch/docker/templates/Dockerfile.j2

```diff
@@ -129,6 +168,9 @@ RUN mkdir /usr/share/supervisor
 
 WORKDIR /usr/share/elasticsearch
 COPY --from=prep_es_files --chown=1000:0 /usr/share/elasticsearch /usr/share/elasticsearch
+COPY --from=prep_es_files /usr/share/elasticsearch/k-NN/buildSrc/libKNNIndexV1_7_3_6.so /usr/lib
```

Comment:
copy the ```libKNNIndexV1_7_3_6.so```  from   /tmp/jni/libKNNIndexV1_7_3_6.so

Reply:
Done